doc/guides/backporting-to-release-lines.md

```diff
@@ -0,0 +1,124 @@
+# How to Backport a Pull Request to a Release Line
+
+## Staging branches
+
+Each release line has a staging branch that the releaser will use as a scratch
+pad while preparing a release. The branch name is formatted as follows:
+`vN.x-staging` where `N` is the major release number.
+
+### Active staging branches
+
+| Release Line | Staging Branch |
+| ------------ | -------------- |
+| `v7.x`       | `v7.x-staging` |
+| `v6.x`       | `v6.x-staging` |
+| `v4.x`       | `v4.x-staging` |
+
+## What needs to be backported?
+
+If a cherry-pick from master does not land cleanly on a staging branch, the
+releaser will mark the pull request with a particular label for that release
+line, specifying to our tooling that this pull request should not be included.
+The releaser will then add a comment that a backport is needed.
+
+## What can be backported?
+
+The "Current" release line (currently v7.x) is much more lenient than the LTS
+release lines in what can be landed. Our LTS release lines
+(currently v4.x and v6.x) require that commits live in a Current release for at
+least 2 weeks before backporting. Please see the [LTS Plan][] for more
```

Comment:
this is a bit confusing in the way it is selectively using / switching backporting + cherry-picking.

Perhaps 

> Our LTS release lines (v4.x and v6.x as of Feb 2017) require that commits mature in a Current release for at least 2 weeks before they can be landed on staging. If the commit can not be cherry-picked from master a manual backport will need to be submitted. Please see the [LTS Plan][] for more information.